src/composition/watchers/watcher/introspection.rs

```diff
@@ -50,8 +50,17 @@ impl SubgraphIntrospection {
             .skip(1)
             .filter_map(|change| async move {
                 match change {
-                    Ok(subgraph) => Some(subgraph),
+                    Ok(subgraph) => {
+                        eprintln!(
+                            "Connectivity restored for subgraph \"{}\".\n",
+                            subgraph.name()
+                        );
+                        Some(subgraph)
+                    }
                     Err(err) => {
+                        eprintln!("Error detected communicating with subgraph: {}", err);
```

Comment:
Might want to use `rover_std::errln` for these: https://github.com/apollographql/rover/blob/main/crates/rover-std/src/print.rs#L30